WebRoot/messages/ZaMsg_hi.properties

```diff
@@ -2706,4 +2706,11 @@ MB_But_Wipe_Cancel = \u0935\u093e\u0907\u092a \u0930\u0926\u094d\u0926 \u0915\u0
 MB_Waiting = \u0909\u092a\u0915\u0930\u0923 \u0915\u0940 \u092a\u094d\u0930\u0924\u0940\u0915\u094d\u0937\u093e
 BUSY_BLOCKING_SYNC_DEVICES = /u0938/u093f/u0902/u0915 /u0915/u093f/u090f /u0917/u090f /u0909/u092a/u0915/u0930/u0923/u094b/u0902 /u0915/u094b /u0905/u0935/u0930/u0941/u0926/u094d/u0927 /u0915/u0930 /u0930/u0939/u093e /u0939/u0948 ...
 BUSY_CANCELLING_WIPE = /u0935/u093e/u0907/u092a /u0911/u092a/u0930/u0947/u0936/u0928 /u0930/u0926/u094d/u0926 /u0915/u0930/u0928/u093e ...
-MB_But_Disconnect = /u0921/u093f/u0938/u094d/u0915/u0928/u0947/u0915/u094d/u091f
\ No newline at end of file
+MB_But_Disconnect = /u0921/u093f/u0938/u094d/u0915/u0928/u0947/u0915/u094d/u091f
+MB_Remove_Account_ACK = \u0932\u0902\u092c\u093f\u0924 \u0916\u093e\u0924\u0947 \u0915\u094b \u0928\u093f\u0915\u093e\u0932\u0947\u0902
```

Comment:
this looks suspicious, why we have two types of slashes
`/u and \u`

Reply:
must been came from while coping from unicode converter